test/values.ts

```diff
@@ -643,6 +643,19 @@ export const mockProviders: FetchProvidersOutput[] = [
       { paymentMethod: PaymentMethod.Card, digitalAsset: 'cusd', returnedAmount: 88, fiatFee: 12 },
     ],
   },
+  {
+    name: 'CoinbasePay',
+    restricted: false,
+    unavailable: false,
+    paymentMethods: [PaymentMethod.Coinbase],
+    url: undefined,
+    logo:
+      'https://firebasestorage.googleapis.com/v0/b/celo-mobile-mainnet.appspot.com/o/images%2FcbPay-button.png?alt=media',
+    logoWide: '',
```

Comment:
I think just re-using the same image as for logo would be preferable to an emptystring here

Reply:
oh just noticed these are test values. still, slightly better.